.devops/code-review-pipelines.yml

```
@@ -11,7 +11,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local
-      ref: main
+      ref: CHK-4261-user-stats-service-service-api-key
```

```suggestion
      ref: main
```
TODO: to be replaced after dependent PR merge